src/watchful/client.py

```diff
@@ -362,17 +366,41 @@ def external(
     :type port: str, optional
     :param scheme: The scheme, either "http" or "https", defaults to "http".
     :type scheme: str, optional
+    :param token: The JWT authorization token, defaults to None.
+    :type token: str, optional
     """
 
     assert scheme in [
         "http",
         "https",
     ], '`scheme` must be either "http" or "https"!'
 
-    global SCHEME, HOST, PORT
+    global SCHEME, HOST, PORT, TOKEN
     SCHEME = scheme
     HOST = host
     PORT = port
+    TOKEN = token
+
+
+def set_token(token: str) -> None:
+    """
+    This function sets the global ``TOKEN`` value.
+
+    :param token: The token.
+    :type token: str
+    """
+
+    global TOKEN
+    TOKEN = token
+
+
+def clear_token() -> None:
+    """
+    This function clears the global ``TOKEN`` value.
+    """
+
+    global TOKEN
+    TOKEN = None
```

Comment:
Can you explain a situation where we would need these two functions, and couldn't just use `external` from above?

Reply:
we could, sure - `client.external(token = None)` but I thought the `set_token` and `clear_token` functions, though simple, were much clearer to the reader. Purely for convenience.

Reply:
As this module is almost entirely for running tests, I'm not too worried about convenience. The reason I bring it up is that I can't see a case where tests would use these, and so I think we can drop.

Reply:
removed